bundles/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/internal/event/IntegraStateEvent.java

```diff
@@ -71,10 +73,10 @@ public BitSet getStateBits(StateType stateType) {
         if (stateType.getStartByte() == 0 && bitsCount == stateBits.size()) {
             return stateBits;
         }
-        return stateBitsMap.computeIfAbsent(stateType, k -> {
+        return requireNonNull(stateBitsMap.computeIfAbsent(stateType, k -> {
```

Comment:
Using anything else besides annotations from the `org.eclipse.jdt.annotation` package will likely cause runtime issues because these packages are considered to be optional:

https://github.com/openhab/openhab-addons/blob/9a5832a2ee82a29210801106dfae9754c03eb98b/pom.xml#L142

I think currently it is only safe to use the `Checks` class in test code.

Reply:
It work without. I wonder why I added it. Also fixed some potential NPE while looking through the binding.